swarm/pss/pss.go

```diff
@@ -472,7 +465,19 @@ func (p *Pss) process(pssmsg *PssMsg, raw bool, prox bool) error {
 	p.executeHandlers(psstopic, payload, from, raw, prox, asymmetric, keyid)
 
 	return nil
+}
+
+// copy all registered handlers for respective topic in order to avoid deadlock
```

Comment:
At its current form this comment does not help me. Either, please, give me more context - like where the deadlock may come from - or just drop it completely. I prefer the latter.

Reply:
?
which other context do you need...?

Reply:
@justelad I said I prefer to drop the comment as I don't think it's necessary to explain the code. I might be wrong, but then I think I miss something, but that I cannot know.